Cargo.toml

```diff
@@ -15,9 +15,5 @@ rand            = "*"
 regex           = "*"
 regex_macros    = "*"
 rustc-serialize = "*"
-# Temporarily use the github time crate, due to a Windowns bug in the released version
-# time            = "*"
-
-[dependencies.time]
-
-git = "https://github.com/rust-lang/time"
\ No newline at end of file
+time            = "*"
```

Comment:
This is not really important at all, but can we please keep the dependencies in alphabetical order. Slight OCD speaking here. ;)